### DISCUSSION TITLE
Heiman HS3AQ Co2 Sensor Test - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#30] [Comments]
# Heiman HS3AQ Co2 Sensor Test - Haade.fr

Heiman Zigbee temperature, humidity, Co2 sensor test and integration in homeassistant

https://haade.fr/en/blog/test-co2-sensor-zigbee-heiman-HS3AQ

**Comment by Faber70** [comment]
Hi Nico, great tutorial. I'm intentionate to buy the HS3AQ, I'm using ZHA protocol for my zigbee gatway. Do you know if the HS3AQ work with ZHA?
 thank you in advanced 
regards Fabrizio
